css/style.css

```diff
@@ -0,0 +1,678 @@
+  width: 13rem;
+  background-color: rgba(255, 255, 255, 0.1);
+  padding: 1rem;
+}
+
+.program .program-title {
+  color: #ec5242;
+  font-weight: 700;
+  font-size: 1.063rem;
+}
+
+.program:hover {
+  border: 0.25rem solid #fff;
+}
+
+.see-the-whole-program p {
+  font-size: 30px;
+}
+
+/* \\\\\\\\Featured Section//////// */
+
+#featured-speakers {
+  display: flex;
+  flex-wrap: wrap;
+  position: relative;
+}
+
+.fs-header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  top: 0;
+  left: 45%;
+}
+
+.fs-content {
+  display: flex;
+  justify-content: center;
+  align-items: baseline;
+  padding-top: 1rem;
+  width: 35%;
+  margin: 1.4rem 7rem 0 7rem;
+  gap: 1rem;
+}
+
+.speaker-content {
+  float: left;
+}
+
+.speaker {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  height: 12rem;
+  transition: 0.35s;
+}
+
+.speaker:hover,
+.speaker:active {
+  padding: 0.8rem;
+}
+
+.speaker .speaker-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin: 0.1rem 0 0 0.5rem;
+}
+
+.speaker .speaker-img {
+  width: 10rem;
+  height: 9rem;
+}
+
+.speaker-content .speaker-title {
+  display: flex;
+  flex-direction: row;
+  font-size: 24px;
+  margin-top: -1rem;
+}
+
+.speaker-content .italic-text {
+  margin-top: -0.5rem;
+  font-style: italic;
+  color: #ec5242;
+}
+
+.speaker-content .speaker-hr {
+  color: #adadad;
+  border-style: solid;
+  border-width: 2px;
+  width: 10%;
+  margin-top: -0.6rem;
+}
+
+.speaker-content .speaker-description {
+  margin-top: -0.6rem;
+}
+
+/* \\\\\\\\Partners Section//////// */
+.partners {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #414246;
+}
+
+.partners-header .partner {
+  color: #d3d3d3;
+  font-weight: 400;
+}
+
+.partners-header .hr-line {
+  margin-left: 25%;
+  width: 38%;
+}
+
+.partners .partner-imgs {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  gap: 2rem;
+  margin-left: 20rem;
+  margin-right: 20rem;
+}
+
+.partner-imgs .partner-img {
+  height: 2.8rem;
+  width: 7.5rem;
+}
+
+/* \\\\\\\\Footer Section//////// */
+footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+}
+
+.footer-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  gap: 3rem;
+}
+
+.footer-container img {
+  margin-left: 12rem;
+  width: 17%;
+}
+
+.footer-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.footer-content .f-c-h {
+  font-weight: 600;
+}
+
+.footer-content p {
+  margin-right: 3rem;
+}
+
+.none {
+  display: block;
+}
+
+.join-cs {
+  display: none;
+}
+
+.non {
+  display: none;
+}
+
+/* From 0 to 768 */
+
+@media only screen and (max-width: 767px) {
+  .top-menu {
+    display: none;
+  }
+
+  .navbar img,
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: please use the `mobile-first` approach but you did use `mobile-first` kindly design first `mobile version` then design the `desktop version` then make changes with  `@media(min-width: 767px)` to change the design from 767px onwards. :slightly_smiling_face: